src/main/scala/no/gdl/bookapi/integration/crowdin/CrowdinClient.scala

```diff
@@ -92,12 +92,12 @@ class CrowdinClient(fromLanguage: String, projectIdentifier: String, projectKey:
     } else {
       val addedFiles = uploadTries.flatMap(_.get.stats.get.files)
       val result = addedFiles.flatMap(file => {
-        chapters.find(chapter => CrowdinUtils.filenameFor(book, chapter) == file.name).map(chapterForFile => {
-          CrowdinFile(Some(chapterForFile.id), FileType.CONTENT, file)
+        chapters.find(chapter => CrowdinUtils.filenameFor(translation, chapter) == file.name).map(chapterForFile => {
+          CrowdinFile(Some(chapterForFile.id), chapterForFile.seqNo, FileType.CONTENT, file)
         })
       })
 
-      if (result.length == addedFiles.length) {
+      if (result.lengthCompare(addedFiles.length) == 0) {
```

Comment:
For å være ærlig, så kom det som et forslag fra IntelliJ. Dokumentasjonen sier følgende:
```
The method as implemented here does not call `length` directly; its running time
is `O(length min len)` instead of `O(length)`. The method should be overwritten
if computing `length` is cheap.
```

I tilfellet her, så er det sikkert like greit å sammenligne length på begge sider, men meh..